Floofbot/Services/EventLoggerService.cs

```diff
@@ -337,6 +361,9 @@ public Task GuildMemberUpdated(SocketGuildUser before, SocketGuildUser after)
             {
                 try
                 {
+                    if (after.IsBot)
```

Comment:
Shouldn't this be placed after the null check?
Also, do we have to check for whether before is a bot, too?

Reply:
Moved it before it. No need because a user cannot change from a normal account to a bot account and vice versa. Before is just the user before they are updated